### ISSUE TITLE
Using a custom ease for tween behaviour crashes 

### ISSUE THREAD

**Issue by Aldenwar** [Scirra/Construct-3-bugs#3668]
## Problem description

 Using custom eases crashes the preview.

## Attach a .c3p

 https://drive.google.com/open?id=1vopvENknm84zUYSSMMDjCaHpm0f0fAtd

## Steps to reproduce

1. Make a custom ease
2. Try to use the custom ease in event and preview, crashes

## Observed result
Crash

## Expected result

No crash

## More details

**Affected browsers/platforms:** 

**First affected release:**  Worked before, 182.4 stable update broke.

## System details

<details><summary>View details</summary>

PASTE HERE

</details>

**Comment by AshleyScirra** [comment]
Your project opens and previews just fine here in both r182.4 and r187.

Assigning to @DiegoScirra but it is unlikely we can do anything about this unless we can reproduce it. Providing essential information like your system details and the exact text of any error messages always helps, if you omit this information it can be much more difficult to resolve the issue.

**Comment by DiegoScirra** [comment]
Can't reproduce this one either, the project previews fine. Tried following the steps from an empty project as well. Both cases work as expected.

@Aldenwar if you can't provide any more details or specific steps to reproduce the crash, I'm going to have to close this one.

**Comment by Aldenwar** [comment]
I have no idea why but now that I restarted my laptop mine doesn't reproduce either on both the bug report project or the actual one! However, if I make a new project and try to use a custom tween, there is still a crash. So maybe try to do that? 

Here's a screenshot of the error message in the console when the crash happens. 

![errormessage](https://user-images.githubusercontent.com/57374755/75156042-eff79300-5719-11ea-8696-9518180d4182.png)

**Comment by AshleyScirra** [comment]
@Aldenwar - can you provide full steps to reproduce the problem from a new empty project in the latest beta release? @DiegoScirra already stated they tried from an empty project and it still worked.

**Comment by DiegoScirra** [comment]
@Aldenwar from your example I can see you changed the name of the custom ease, I suspect the issue has something to do with that, but I can't pinpoint exactly how it is related.

Can you include when and how you are changing the name in the reproduction steps?

**Comment by Aldenwar** [comment]
Seems to work on latest beta. For the previous version my steps in exact order were: add a sprite, give it tween behaviour, create and immediately rename custom ease, edit the ease, add an event that activates tween with the custom ease. 

Trying it out on a new session with incognito tab I can't reproduce on the current stable version either, so it is quite weird. If I had to guess it might have something to do with having had multiple projects open on the same session, as the first time I noticed the bug was when I had two projects open at once at school computer. No tweens between projects shared similar names though. Couldn't figure out how to reproduce with this.

Since it does work without issues on latest beta I suppose something has fixed it though.
